.dockerignore

```diff
@@ -0,0 +1,6 @@
+**/bin/
```

Comment:
The sample project uses NMica to autogenerate dockerfile on build and dockerignore if it's missing. Both can actually be added to .gitignore since they are autogenerated